packages/react-interactions/events/src/dom/__tests__/FocusWithin-test.internal.js

```diff
@@ -258,6 +269,18 @@ describe.each(table)('FocusWithin responder', hasPointerEvents => {
       expect(onFocusWithinVisibleChange).toHaveBeenCalledTimes(2);
       expect(onFocusWithinVisibleChange).toHaveBeenCalledWith(false);
     });
+
+    it('is called after a focused element is unmounted', () => {
+      const inner = innerRef.current;
+      const target = createEventTarget(inner);
+      target.keydown({key: 'Tab'});
+      inner.focus();
```

Comment:
```suggestion
      const target = createEventTarget(innerRef.current);
      target.keydown({key: 'Tab'});
      target.focus();
```

Reply:
The changes to use `target.focus()` fail the tests, any idea why?

Reply:
Maybe because it is just dispatching an event and not updating the value of `document.activeElement`? Perhaps jsdom does that behind the scenes when you use programmatic `.focus()`